src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,8 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+import mate.academy.model.User;
+
+public class AuthenticationService extends User {
```

Comment:
Can we say that a service that checks the email & password IS A user?
User class represents some actual existing user with some info that we need to know about him. AuthenticationService should be treated as some office building for example, that has a mechanism to authenticate those users(`login()` method), so those are 2 different things and can not be mixed together because a building is not a person.

For now here are some links to read more about the [inheritance](https://www.geeksforgeeks.org/inheritance-in-java/), [composition vs inheritance](https://www.digitalocean.com/community/tutorials/composition-vs-inheritance), do not dig too deep, because we will cover those things in the course